app/Models/ProfileBanner.php

```diff
@@ -42,4 +43,44 @@ public function country()
     {
         return $this->belongsTo(Country::class, 'country_acronym');
     }
+
+    public function macroVisible()
```

Comment:
Maybe call this `current` or something instead of `visible`? Kinda ambiguous.

Reply:
Went with `active` because it might be for either current or previous tournament.